python/cuml/fil/fil.pyx

```diff
@@ -265,6 +266,30 @@ cdef class ForestInference_impl():
                       &treelite_params)
         return self
 
+    def load_from_treelite_model2(self,
```

Comment:
looks identical to load_from_treelite_model to me... but maybe I'm just missing it?

Reply:
Refactored.